EngineLayer/ProteinParsimony/ProteinGroup.cs

```diff
@@ -18,7 +18,7 @@ public ProteinGroup(HashSet<Protein> proteins, HashSet<PeptideWithSetModificatio
             AllPeptides = peptides;
             UniquePeptides = uniquePeptides;
             AllPsmsBelowOnePercentFDR = new HashSet<PeptideSpectralMatch>();
-            SequenceCoveragePercent = new List<double>();
+            SequenceCoverageFraction = new List<double>();
```

Comment:
Just curious as to why this is changed over from percent to fraction?

Reply:
Anthony mad this request specifically. I'm agnostic about it but it helps him. https://github.com/smith-chem-wisc/MetaMorpheus/issues/1688